Scripts/assignment.py

```diff
@@ -365,6 +335,78 @@ def assign_transit(self, scen_id):
                              # scenario=scenario)
         matrix_results = self.emme_modeller.tool(
             "inro.emme.transit_assignment.extended.matrix_results")
-        matrix_results(param.result_spec, scenario)
+        # Transit assignment result specification
+        mtx = param.emme_mtx
+        result_spec = {
+            "type": "EXTENDED_TRANSIT_MATRIX_RESULTS",
+            "total_impedance": mtx["time"]["transit"]["id"],
+            "actual_first_waiting_times": mtx["transit"]["fw_time"]["id"],
+            "actual_total_waiting_times": mtx["transit"]["tw_time"]["id"],
+            "by_mode_subset": {
+                "modes": param.transit_modes,
+                "distance": mtx["dist"]["transit"]["id"],
+                "avg_boardings": mtx["transit"]["num_board"]["id"],
+                "actual_total_boarding_times": mtx["transit"]["board_time"]["id"],
+                "actual_in_vehicle_times": mtx["transit"]["inv_time"]["id"],
+                "actual_aux_transit_times": mtx["transit"]["aux_time"]["id"],
+            },
+        }
+        matrix_results(result_spec, scenario)
         self.logger.info("Transit assignment performed for scenario " 
-                        + str(scen_id))
\ No newline at end of file
+                        + str(scen_id))
+        
+class Car:
```

Comment:
We could put these classes to separate .py files so they'd be more easy to find. Also in general case sometimes this can improve the decoupling of classes so that accidentally we don't access something that should be considered private.